### PR TITLE
introducing global phase

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: _posts/2023-05-08-improving-llm-context-ranking-with-cross-encoders.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+phases, where more efficient retrieval and ranking models have
+significantly pruned the number of documents in advance.
+
+
+## Phased ranking
+
+Vespa has best-in-class support for expressing [multi-phased retrieval
+and ranking](https://docs.vespa.ai/en/phased-ranking.html). Using
+multi-stage retrieval and ranking pipelines is an industry best
+practice for _efficiently _matching and ranking content. The basic
```
Comment (kkraune) [inline _posts/2023-05-08-improving-llm-context-ranking-with-cross-encoders.md:61]:
```suggestion
practice for _efficiently_ matching and ranking content. The basic
```

File: _posts/2023-05-08-improving-llm-context-ranking-with-cross-encoders.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+<pre>
+rank-profile global-phased {
+  first-phase {
+    expression: log(bm25(title)) + cos(distance(field,embedding))    
+  }
+  second-phase {
+    expression { firstPhase + lightgbm("f834_v2.json")}
+    rerank-count: 1000
+  }
+   global-phase {
```
Comment (kkraune) [inline _posts/2023-05-08-improving-llm-context-ranking-with-cross-encoders.md:125]:
```suggestion
  global-phase {
```

**Comment by bjorncs** [comment]
PR build fails
```
09:23:08 For the Links > External check, the following failures were found:
09:23:08 
09:23:08 * At _site/improving-llm-context-ranking-with-cross-encoders/index.html:448:
09:23:08 
09:23:08   External link https://slack.vespa.ai/ failed: got a time out (response code 0) (status code 0)
09:23:08 
09:23:08 * At _site/latest.html:19:
09:23:08 
09:23:08   External link https://blog.vespa.ai/assets/2023-05-08-improving-llm-context-ranking-with-cross-encoders/will-van-wingerden-dsvJgiBJTOs-unsplash.jpg failed (status code 404)
09:23:08 
09:23:08 * At _site/latest.html:161:
09:23:08 
09:23:08   External link https://blog.vespa.ai/assets/2023-05-08-improving-llm-context-ranking-with-cross-encoders/image1.png failed (status code 404)
09:23:08 
09:23:08 * At _site/latest.html:205:
09:23:08 
09:23:08   External link https://slack.vespa.ai/ failed: got a time out (response code 0) (status code 0)
09:23:08 
09:23:08 * At _site/securing-vespa-with-mutually-authenticated-tls/index.html:302:
09:23:08 
09:23:08   External link https://datatracker.ietf.org/doc/html/rfc2818#section-3 failed (status code 429)
09:23:08 
09:23:08 
09:23:08 HTML-Proofer found 5 failures!
```

**Review by kkraune — APPROVED** [review]
Awesome!

**Review by bjorncs — APPROVED** [review]
🥇

**Review by bjorncs — APPROVED** [review]
🥇

**Review by bjorncs — APPROVED** [review]
🥇

**Review by bjorncs — APPROVED** [review]
🥇

**Review by kkraune — APPROVED** [review]

**Pull request merged**